ripple/src/main/test/script.test.js

```diff
@@ -123,8 +123,8 @@ test('searchInput: reads and searches user input', () => {
   var event = new KeyboardEvent('keyup', {'keyCode': 13});
   document.dispatchEvent(event);
   scriptsModule.searchInput(event);
-  expect(localStorage.getItem("galleryPageSearchTag")).toEqual("Chinese");
-  expect(localStorage.getItem("galleryPageName")).toEqual("'Chinese'");
+  expect(localStorage.getItem("galleryPageSearchTag")).toBe("Chinese");
```

Comment:
Just curious, what's the difference between toEqual() vs toBe()?

Reply:
toBe is a stricter comparison that returns false if the two objects are not the exact same in memory. It's equivalent to the '===' comparator. toEqual is more similar to '==' comparator. So objectOne === objectTwo returns false for toBe() but true for toEqual()